Sources/StringsLintFramework/Parsers/StructuredPlaceholderCommentParser.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mark Hall on 2022-07-19.
+//
+
+import Foundation
+
+public struct StructuredPlaceholderCommentParser: LocalizableParser {
+
+    public static var identifier: String {
+        return "structured_placeholder_comment_parser"
+    }
+
+    public var supportedFileExtentions: [String] {
+        return [ "strings" ]
+    }
+
+    public init() {
+    }
+
+    public init(configuration: Any) throws {
+        // Parser does not support any configuration
+        self.init()
+    }
+
+    public func support(file: File) -> Bool {
```

Comment:
We can specify a configuration file and tell it which path to lint https://github.com/dral3x/StringsLint#configuration I'm thinking that is the easiest way to do this. That means that _all_ the `StringsLint` rules will only be applied to the english file - which I'm pretty sure is what we want anyway since the other ones are just generated based on the english one. @rlnucci can you confirm my assumption is correct?

Reply:
yes, that is correct